spec/analyses/urlStopwordSpec.js

```diff
@@ -0,0 +1,10 @@
+var urlStopwords = require( "../../js/analyses/checkUrlForStopwords.js" );
+
+describe( "Checks the URL for stopwords", function(){
+	it("returns any stopwords found", function(){
+		expect( urlStopwords( "before-and-after" ) ).toContain( "before" );
```

Comment:
You're only checking if the array contains these things but not if the array actually only has these values. `urlStopwords` could return a gigantic array with tons of words including `before` and `after` and these tests would still pass.

So this should probably be `toBe`.